spec/README.md

```diff
@@ -0,0 +1,18 @@
+# Running Mongoid Tests
+
+## Overview
+### Quick Start
+Spin up a MongoDB deployment against which to run the Mongoid specs. Mongoid specs support a variety of MongoDB topologies, but the simplest is a single MongoDB instance:
+
+    # Launch mongod in one terminal
+    mkdir /tmp/mdb
+    mongod --dbpath /tmp/mdb
+
+Run the test suite in a separate terminal:
+
+    rake
+
+
+## Caveats
+### "Too many open files" error
+On MacOS, you may encounter a "Too many open files" error when running the tests. If this happens, run the command `ulimit -n 10000` to increase the number of files that can be opened and then re-run the tests.
```

Comment:
Did this apply to the test suite or to the server?

Reply:
Good point. Will clarify!